survey_runner.tf

```diff
@@ -0,0 +1,190 @@
+resource "aws_alb_target_group" "survey_runner" {
+  name     = "${var.env}-survey-runner-ecs"
+  port     = 80
+  protocol = "HTTP"
+  vpc_id   = "${var.vpc_id}"
+
+  health_check = {
+    healthy_threshold   = 2
+    unhealthy_threshold = 2
+    interval            = 5
+    timeout             = 2
+    path                = "/status"
+  }
+
+  tags {
+    Environment = "${var.env}"
+  }
+}
+
+resource "aws_alb_listener_rule" "survey_runner" {
+  listener_arn = "${var.aws_alb_listener_arn}"
+  priority     = 10
+
+  action {
+    type             = "forward"
+    target_group_arn = "${aws_alb_target_group.survey_runner.arn}"
+  }
+
+  condition {
+    field  = "host-header"
+    values = ["${aws_route53_record.survey_runner.name}"]
+  }
+}
+
+resource "aws_route53_record" "survey_runner" {
+  zone_id = "${var.dns_zone_id}"
+  name    = "${var.env}-surveys.${var.dns_zone_name}"
+  type    = "CNAME"
+  ttl     = "60"
+  records = ["${var.aws_alb_dns_name}"]
+}
+
+data "template_file" "survey_runner" {
+  template = "${file("${path.module}/task-definitions/survey-runner.json")}"
+
+  vars {
+    EQ_RABBITMQ_HOST                     = "${var.rabbitmq_ip_prime}"
+    EQ_RABBITMQ_HOST_SECONDARY           = "${var.rabbitmq_ip_failover}"
+    EQ_RABBITMQ_QUEUE_NAME               = "${var.message_queue_name}"
+    EQ_SERVER_SIDE_STORAGE_DATABASE_HOST = "${var.database_address}"
+    EQ_SERVER_SIDE_STORAGE_DATABASE_PORT = "${var.database_port}"
+    EQ_SERVER_SIDE_STORAGE_DATABASE_NAME = "${var.database_name}"
+    EQ_LOG_LEVEL                         = "${var.eq_log_level}"
+    AWS_DEFAULT_REGION                   = "${var.aws_default_region}"
+    EQ_UA_ID                             = "${var.google_analytics_code}"
+    EQ_SECRETS_FILE                      = "${var.secrets_file_name}"
+    LOG_GROUP                            = "${aws_cloudwatch_log_group.survey_runner.name}"
+    CONTAINER_REGISTRY                   = "${var.docker_registry}"
```

Comment:
I didn't make them EQ because they are more about the environment than the application so it gave a logical split